internal/cli/apps.go

```diff
@@ -638,8 +659,13 @@ func updateAppCmd(cli *cli) *cobra.Command {
 				}
 			}
 
+			clientMetadata := make(map[string]interface{}, len(inputs.Metadata))
+			for k, v := range inputs.Metadata {
+				clientMetadata[k] = v
+			}
+
 			// Load updated values into a fresh app instance
-			a := &management.Client{}
+			a := &management.Client{ClientMetadata: &clientMetadata}
```

Comment:
One thing to verify here is if a user doesn't specify metadata when updating, effectively skipping it, does that overwrite any previously existing metadata on remote?

Reply:
Changed appropriately. Great catch!